operators/pkg/controller/elasticsearch/sset/list.go

```diff
@@ -61,6 +67,24 @@ func (l StatefulSetList) PodNames() []string {
 	return names
 }
 
+// GetActualPods returns the list of pods currently existing in the StatefulSetList.
+// Some "expected" pods may be missing from the returned list if they don't exist yet
+// or are temporarily down.
```

Comment:
"down" to mean "deleted but not yet re-created" and/or "going to be created soon"?

I'm not sure the comment around expected != actual is worthwhile at all.